po/MusicBlocks.pot

```diff
@@ -636,74 +636,74 @@ msgstr ""
 #: js/analytics.js:316
 #: js/basicblocks.js:341
 #: js/basicblocks.js:354
-msgid "rhythm"
+msgid "Rhythm"
```

Comment:
The "rhythm" sting is used in more places than just turtledefs.js so if we change it here, we need to either change it everywhere -- not recommended -- or add a new entry for "Rhythm". The same is true for most of the other strings you changed.

I will run a script to generate a new POT file directly from the source code and another script to update the PO flles.